tests/test_lock.py

```diff
@@ -66,4 +66,21 @@ def test_cache_lock(self):
         thread1.join()
         thread2.join()
 
+    def test_redis_lock(self):
```

Comment:
3d party dependent and long-running tests should be optional (ex: test_contrib_celery)